sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Azure.AI.Language.QuestionAnswering.csproj

```diff
@@ -7,6 +7,7 @@
     <ApiCompatVersion>1.0.0</ApiCompatVersion>
     <PackageTags>Azure AI Language Question Answering QnA Maker</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <DefineConstants>$(DefineConstants);EXPERIMENTAL</DefineConstants>
```

Comment:
What's `EXPERIMENTAL` for? I don't see it referenced in a browser-based search within this PR (with all file content rendered).

Reply:
The effect of having this is that it activates experimental Azure.Core functions that are used by the LLC. Without it, I get errors about the following functions not being found:
 
- `LowLevelOperationHelpers.ProcessMessage()` + async version.
- An overload of the `_pipeline.ProcessMessage()` + async version.

I found this Constant being included in the [Azure.Template.LLC.csproj](https://github.com/Azure/azure-sdk-for-net/blob/117081fc19582e81a0f5be0f2607c77f83f0c9fe/sdk/template-LLC/Azure.Template.LLC/src/Azure.Template.LLC.csproj#L10). I also found it being included in a similar way in other SDK projects including Synapse, WebPubSub and AgriFood.

Is this not the correct solution?

Reply:
If that's what it takes, seems correct. There's no final documentation for LLC yet that I'm aware.